containers/filters/editor/EditConditionValue.js

```diff
@@ -1,24 +1,16 @@
-import React, { useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
 import { Input, Button } from 'react-components';
 import { noop } from 'proton-shared/lib/helpers/function';
 
 function EditConditionValue({ value, onClickDelete, onEdit }) {
-    const [state, setState] = useState(value);
-
     const handleClick = () => onClickDelete(value);
-    const handleInput = ({ target }) => setState(target.value);
-    const handleKeyUp = ({ key }) => {
```

Comment:
Where did you move this logic?

Reply:
When editing, there is an input field with a button `Delete`, in contrast to input+`Add` when creating filters. When creating filter, pressing enter adds a condition, but when editing, it submits the whole form.

Not sure about what the behavior should be, but I removed the onKeyUp, as it did nothing in "edit" case. Now the model value updates on every change in the input field.